code/game/atoms.dm

```diff
@@ -666,6 +666,14 @@
 	SEND_SIGNAL(src, COMSIG_ATOM_RCD_ACT, user, the_rcd, passed_mode)
 	return FALSE
 
+/**
+  * Respond to our atom being teleported
+  *
+  * Default behaviour is to send COMSIG_ATOM_TELEPORT_ACT and return
+  */
+/atom/proc/teleport_act()
+	SEND_SIGNAL(src,COMSIG_ATOM_TELEPORT_ACT)
```

Comment:
Why have a signal for it if you never use it?

Reply:
for future use, mostly. also, just about every other _act proc has that line, and most dont seem to use it